File: 백준/파이썬/1차원 배열/평균은 넘겠지.py
# 4344
# 평균은 넘겠지
# Python
# 각 케이스마다 한 줄씩 평균을 넘는 학생들의 비율을 반올림하여 소수점 셋째 자리까지 출력하시오.

C = int(input())
rate = []

for i in range(C):
    score = list(map(int, input().split()))
    avg = sum(score[1:]) / score[0]
    over = 0
    for i in range(score[0] + 1):
        if score[i] > avg:
            over += 1
    rate.append(over / score[0] * 100)

for i in range(C):
    print("%.3f%%" % (rate[i]))

File: 백준/C언어/평균은 넘겠지.c
// 4344
// 평균은 넘겠지
// C언어
// 각 케이스마다 한 줄씩 평균을 넘는 학생들의 비율을 반올림하여 소수점 셋째 자리까지 출력하시오.

#include <stdio.h>

int main(void)
{
	int C;
	int N;
	int score[1000];
	int sum;
	int avg;
	int over;
	double rate[100];

	scanf("%d", &C);

	for (int i = 0; i < C; i++)
	{
		scanf("%d", &N);

		sum = 0, avg = 0, over = 0;

		for (int j = 0; j < N; j++)
		{
			scanf("%d", &score[j]);
			sum += score[j];
		}
		avg = (double)sum / N;
		for (int k = 0; k < N; k++)
		{
			if (score[k] > avg)
				over++;
		}
		rate[i] = ((double)over / N) * 100;
	}
	for (int l = 0; l < C; l++)
	{
		printf("%.3lf%%\n", rate[l]);
	}

	return 0;
}